charts/splunk-connect-for-snmp/values.schema.json

```diff
@@ -0,0 +1,613 @@
+            },
+            "name": {
+              "type": "string"
+            }
+          }
+        },
+        "service": {
+          "type": "object",
+          "properties": {
+            "annotations": {
+              "type": "object"
+            }
+          }
+        },
+        "resources": {
+          "type": "object"
+        },
+        "nodeSelector": {
+          "type": "object"
+        },
+        "tolerations": {
+          "type": "array"
+        }
+      }
+    },
+    "traps": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "replicaCount": {
+          "type": "integer"
+        },
+        "usernameSecrets": {
+          "type": "array"
+        },
+        "securityEngineId": {
+          "type": "array"
+        },
+        "aggregateTrapsEvents": {
+          "type": ["string", "boolean"]
+        },
+        "communities": {
+          "type": "object"
+        },
+        "service": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "annotations": {
+              "type": "object"
+            },
+            "usemetallb": {
+              "type": "boolean"
+            },
+            "metallbsharingkey": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string"
+            },
+            "port": {
+              "type": "integer"
+            },
+            "nodePort": {
+              "type": "integer"
+            }
+          }
+        },
+        "loadBalancerIP": {
+          "type": "string"
+        },
+        "resources": {
+          "type": "object"
+        },
+        "autoscaling": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "enabled": {
+              "type": "boolean"
+            },
+            "minReplicas": {
+              "type": "integer"
+            },
+            "maxReplicas": {
+              "type": "integer"
+            },
+            "targetCPUUtilizationPercentage": {
+              "type": "integer"
+            }
+          }
+        },
+        "logLevel": {
+          "type": "string"
+        },
+        "nodeSelector": {
+          "type": "object"
+        },
+        "tolerations": {
+          "type": "array"
+        },
+        "podAntiAffinity": {
+          "type": "string"
+        }
+      }
+    },
+    "serviceAccount": {
+      "type": "object",
+      "properties": {
+        "create": {
+          "type": "boolean"
+        },
+        "annotations": {
+          "type": "object"
+        },
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    "useDeprecatedAPI": {
+      "type": "boolean"
+    },
+    "commonAnnotations": {
+      "type": "object"
+    },
+    "mongodb": {
+      "type": "object"
+    },
+    "redis": {
+      "type": "object"
+    },
+    "mibserver": {
+      "type": "object"
```

Comment:
I think I would just allow to configure repository, pullPolicy and tag for mibserver